rss2017/python/assign-reviewers.py

```diff
@@ -145,6 +145,17 @@ def get_reviewer_group(reviewer, paper_number, conflict_list):
     client.add_members_to_group(reviewers,new_reviewer_id)
     reviewers_nonreaders = client.get_group(TRACK+'/Paper'+paper_number+'/Reviewers/NonReaders')
     client.add_members_to_group(reviewers_nonreaders, new_reviewer_id)
+
+    ## the Poster track submissions are only visible to the author, co-chairs
+    ## and the reviewers assigned to that paper.
+    ## Here we add the reviewers to readers list to see the paper.
+    if TRACK == POSTER:
+        notes = [note for note in submissions if str(note.number) == str(paper_number)]
+        if not (reviewer in note.readers):
+            note.readers.append(reviewer)
```

Comment:
what is reviewer? the AnonGroup? the tilde group? or the group that holds all the assigned reviewer of the paper:

```
roboticsfoundation.org/RSS/2017/Workshop/Proceedings/Paper1/Reviewers
```

I think you should use this latest one.